fakesome.js

```diff
@@ -2669,6 +2669,8 @@ Color.prototype = {
       return string.hslString(this.values.hsl, this.values.alpha);
    },
    hslaString: function() {
+	   
+	   
```

Comment:
You added 2 useless empty lines here @qualitynonsense . You should review your changes more carefully in the future before opening the merge request. 😉